lib/auth/middleware.go

```diff
@@ -383,9 +383,9 @@ func getCustomRate(endpoint string) *ratelimit.RateSet {
 		return rates
 	// Passwordless RPCs (potential unauthenticated challenge generation).
 	case "/proto.AuthService/CreateAuthenticateChallenge":
-		const period = defaults.LimiterPasswordlessPeriod
-		const average = defaults.LimiterPasswordlessAverage
-		const burst = defaults.LimiterPasswordlessBurst
+		const period = defaults.LimiterPeriod
```

Comment:
Is there any reason not to also use these defaults for the account recovery RPCs above?

As a sidenote, it looks like the changes in this PR are exclusive to HTTP endpoints, and not RPCs. Might be worth clarifying in the PR description / commit message.

Reply:
> Is there any reason not to also use these defaults for the account recovery RPCs above?

I don't see any reason to increase those rates, do you?

> As a sidenote, it looks like the changes in this PR are exclusive to HTTP endpoints

Good point, I will update the descriptions